packages/gatsby-worker/src/__tests__/integration.ts

```diff
@@ -1,9 +1,16 @@
 import "jest-extended"
 import { WorkerPool } from "../"
 import { isPromise } from "../utils"
+import { MessagesFromChild, MessagesFromParent } from "./fixtures/test-child"
 
 describe(`gatsby-worker`, () => {
-  let workerPool: WorkerPool<typeof import("./fixtures/test-child")> | undefined
+  let workerPool:
+    | WorkerPool<
```

Comment:
To keep things DRY we could move this into its own type and then reuse it in line 24 with `workerPool: YourType`

Reply:
https://github.com/gatsbyjs/gatsby/pull/32159/commits/0623c2c2f4ac1a74e80f7ee787a4c785cab520d8 got rid of generics when constructing - it seemed to be fine with that (tho my IDE started complaining about `workerPool` being undefined which it didn't before